doc-content/enterprise-only/installation/clustering-datagrid-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='clustering-datagrid-proc_{context}']
+= Installing and configuring {DATAGRID}
+
+To use {DATAGRID} for more efficient searching across cluster nodes, install and configure {DATAGRID} for the {PRODUCT} clustered environment. Use the following instructions to configure a simplified, non-high availability environment.
+
+For information about {DATAGRID} modules for {EAP}, see
+https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html-single/red_hat_data_grid_user_guide/index#red_hat_data_grid_modules_for_eap[{DATAGRID} modules for EAP] in the _{DATAGRID} User Guide_.
+
+
+.Prerequisites
+* A Java Virtual Machine (JVM) environment compatible with Java 8.0 or later is installed.
+* A backed-up {EAP} installation version 7.2 or higher is available. The base directory of the {EAP} installation is referred to as `__EAP_HOME__`.
+* {PRODUCT} is installed and configured.
+* Sufficient user permissions to complete the installation are granted.
+
+.Procedure
+. Login to the Red Hat Customer Portal and navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
```

Comment:
Log in to the...

Actually, delete this:
Login to the Red Hat Customer Portal and
So you have:
Navigate to the ...

Reply:
Done